### ISSUE TITLE
XML::LibXML has by default enabled insecure http network access

### ISSUE THREAD

**Issue by pali** [shlomif/perl-XML-LibXML#43]
XML::LibXML may be instructed to download some parts of XML even when `expand_entities` and `load_ext_dtd` are disabled as happened in https://github.com/shlomif/perl-XML-LibXML/pull/39

So network access which is enabled by default is still a security problem even after avoiding XXE.

Moreover https network access is not supported. Only http which is another security issue in network access.

**Comment by toddr** [comment]
I tend to agree, even if it's a breaking change, that maybe we should consider turning off network access by default. People need to explicitly enable the scary, unsafe thing.

**Comment by toddr** [comment]
@toddr-bot plan

**Comment by toddr** [comment]
@toddr-bot what about asking with EUMM's prompt if the user wants to enable secure defaults by setting no_network => 1 (THIS MAY BREAK EXISTING MODULES AND SOFTWARE THAT ASSUMES network is on by default)
